### ISSUE TITLE
Atividades

### ISSUE THREAD

**Issue by JoaoMatheusHneda** [JoaoMatheusHneda/JoaoMatheusHneda.github.io#1]
<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />

<meta name="author" content="João Matheus S. K. T. Hneda" />

<title>Exercícios de Variáveis Aleatórias Bidimensionais</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/journal.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/navigation-1.1/codefolding.js"></script>
<link href="site_libs/highlightjs-1.1/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-1.1/highlight.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs && document.readyState && document.readyState === "complete") {
   window.setTimeout(function() {
      hljs.initHighlighting();
   }, 0);
}
</script>

<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>

</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>

<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 61px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 66px;
  margin-top: -66px;
}

.section h2 {
  padding-top: 66px;
  margin-top: -66px;
}
.section h3 {
  padding-top: 66px;
  margin-top: -66px;
}
.section h4 {
  padding-top: 66px;
  margin-top: -66px;
}
.section h5 {
  padding-top: 66px;
  margin-top: -66px;
}
.section h6 {
  padding-top: 66px;
  margin-top: -66px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<div class="container-fluid main-container">

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>
<script>
$(document).ready(function () {
  window.initializeCodeFolding("hide" === "show");
});
</script>

<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
  padding-left: 25px;
  text-indent: 0;
}

.tocify .list-group-item {
  border-radius: 0px;
}

</style>

<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">

<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html"></a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Primeira Atividade</a>
</li>
<li>
  <a href="teste.html">Segunda Atividade</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div>
  </div>
</div>

<div class="fluid-row" id="header">

<div class="btn-group pull-right">
<button type="button" class="btn btn-default btn-xs dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
</ul>
</div>

<h1 class="title toc-ignore">Exercícios de Variáveis Aleatórias Bidimensionais</h1>
<h4 class="author"><em>João Matheus S. K. T. Hneda</em></h4>
<h4 class="date"><em>11 de Maio de 2017</em></h4>

</div>

<div id="exercicio-1" class="section level1">
<h1><span class="header-section-number">1</span> Exercício 1)</h1>
<div id="defeitos-em-carros" class="section level2">
<h2><span class="header-section-number">1.1</span> Defeitos em carros</h2>
<p>Os carros de uma determinada marca podem apresentar dois tipos de defeitos até a primeira revisão: defeitos graves (que comprometem o funcionamento) e defeitos menores (tais como defeitos de acabamento, lâmpadas queimadas, etc). Suponha que costumam ocorrer até 2 defeitos graves e até 3 menores.</p>
<p>Sejam X a v.a que representa o número de defeitos graves e Y a v.a. representando o número de defeitos menores de um carro sorteado ao acaso. A Tabela abaixo mostra como se distribuem as probabilidades conjuntas p (xi , yj) para os diferentes valores de X e Y . Os valores das linhas e colunas correspondem aos valores de X e Y, respectivamente. Note que a soma de todas as probabilidades é igual a 1. Calcule no R a probabilidade de ocorrerem mais defeitos graves do que defeitos menores e também calcule a probabilidade de ocorrer o mesmo número de defeitos graves e defeitos menores.</p>
<table>
<thead>
<tr class="header">
<th>X↓ Y↑</th>
<th>0</th>
<th>1</th>
<th>2</th>
<th>3</th>
<th>P(X=xi)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>0</td>
<td>0.20</td>
<td>0.20</td>
<td>0.14</td>
<td>0.06</td>
<td>0.60</td>
</tr>
<tr class="even">
<td>1</td>
<td>0.15</td>
<td>0.08</td>
<td>0.04</td>
<td>0.03</td>
<td>0.30</td>
</tr>
<tr class="odd">
<td>2</td>
<td>0.05</td>
<td>0.02</td>
<td>0.02</td>
<td>0.01</td>
<td>0.10</td>
</tr>
<tr class="even">
<td>P(Y=yj)</td>
<td>0.40</td>
<td>0.30</td>
<td>0.20</td>
<td>0.10</td>
<td>1</td>
</tr>
</tbody>
</table>
<p><strong>Resolução:</strong></p>
<pre class="r"><code>x &lt;- data.frame(&quot;0&quot;=c(0.20,0.15,0.05),&quot;1&quot;=c(0.20,0.08,0.02),&quot;2&quot;=c(0.14,0.04,0.02),&quot;3&quot;=c(0.06,0.03,0.01))
colnames(x) &lt;- c(&quot;0&quot;,&quot;1&quot;,&quot;2&quot;,&quot;3&quot;)
rownames(x) &lt;- 0:2
a &lt;- x[2,1]+x[3,1]+x[3,2]
b &lt;- x[1,1]+x[2,2]+x[3,3]</code></pre>
<p>A probabilidade de ocorrerem mais defeitos graves do que defeitos menores é igual a 0.22 e a probabilidade de ocorrer o mesmo número de defeitos graves e defeitos menores corresponde a 0.3. ______________________________________________________________________</p>
</div>
</div>
<div id="exercicio-2" class="section level1">
<h1><span class="header-section-number">2</span> Exercício 2)</h1>
<div id="concurso-publico" class="section level2">
<h2><span class="header-section-number">2.1</span> Concurso Público</h2>
<p>Num concurso público para engenheiros, a prova de conhecimentos consta de uma parte teórica e uma parte prática, que devem ser feitas nesta ordem. Cada candidato só tem acesso ao enunciado da parte prática após ter terminado e entregado a parte teórica. O prazo máximo é de 2 horas para completar a prova como um todo. Sejam Y o tempo gasto para completar a parte teórica, e X o tempo total gasto para completar toda a prova, ambos medidos em horas. Admita que o vetor aleatório (X,Y) têm uma função de densidade conjunta dada pela expressão</p>
<p><span class="math display">\[ f(x,y) = \begin{cases} \frac{1}{2}xy &amp; \text{ se } 0 \leq y \leq x \leq 2 \\ 0 &amp; \text{ c.c } \end{cases} \]</span></p>
<div class="figure">
<img src="http://i.imgur.com/Zjv8qYD.png" alt="Os tempos da parte teórica (y) e total (x) para realizar a prova" />
<p class="caption">Os tempos da parte teórica (y) e total (x) para realizar a prova</p>
</div>
<p>Sabendo disso calcule a probabilidade de que um determinado candidato termine a prova toda em no máximo 1 hora.</p>
<p><strong>Resolução:</strong> Isto equivale a determinar a probabilidade na região R dada por</p>
<p><span class="math display">\[ R = \left \{ (x,y)\in \mathbb{R}^2: 0 \leq y \leq x \leq 1\right \} \]</span></p>
<p>Então,</p>
<p><span class="math display">\[ P[(X,Y)\in R] = \int \int_{R}^{ } f(x,y) dxdy = \int_{0}^{1}\int_{0}^{x} \frac{1}{2} xy \, dydx = 0,0625 \]</span> Logo, podemos concluir que em somente 6,25% dos casos a prova como um todo é entregue em no máximo 1 hora. ______________________________________________________________________</p>
</div>
</div>

</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});

</script>

<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
